src/libnl3/patch/0001-parse-multipath-infinite-loop-fix.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/lib/route/route_obj.c b/lib/route/route_obj.c
```

Comment:
are they upstreamed? can you format the patch properly, author, description, bla, bla

Reply:
The patch files have been reformatted as requested.  A PR has been created to upstream libnl3 changes, but there have been no PRs merged in that project in 2021, so these patches remain necessary.